esphome/components/ble_client/__init__.py

```diff
@@ -0,0 +1,86 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import esp32_ble_tracker
+from esphome.const import (
+    CONF_ID,
+    CONF_MAC_ADDRESS,
+    CONF_NAME,
+    CONF_ON_CONNECT,
+    CONF_ON_DISCONNECT,
+    CONF_TRIGGER_ID,
+)
+from esphome.core import coroutine
+from esphome import automation
+
+DEPENDENCIES = ["esp32_ble_tracker"]
```

Comment:
Please add yourself as a codeowner and run `script/build_codeowners.py`
```suggestion
CODEOWNERS = ["@buxtronix"]
DEPENDENCIES = ["esp32_ble_tracker"]
```